### DISCUSSION TITLE
Is blanket available for macOS?

### DISCUSSION THREAD

**Discussion by vitaminace33** [rafaelmardojai/blanket#123] [Q&A] [answered]
Is there a way to install blanket on macOS through [flathub](https://flathub.org), [homebrew](https://brew.sh), [macports](https://www.macports.org) or a combination of these?

**Comment by rafaelmardojai** [comment, marked as answer]
Sorry for late response. Although should be technically possible, currently it's not available for macOS.

**Reply by vitaminace33** [reply]
Thx, good to know. Let's hope it does come soon, so fanboys can enjoy it.
